lib/test_track.rb

```diff
@@ -52,6 +54,23 @@ def misconfiguration_notifier_class_name=(notifier_class_name)
       @misconfiguration_notifier_class_name = notifier_class_name
     end
 
+    def load_build_timestamp
+      if Rails.env.test? || Rails.env.development?
+        @build_timestamp = Time.zone.now.iso8601
+      elsif File.exist?(BUILD_TIMESTAMP_FILE_PATH)
+        raise "TestTrack's build_timestamp is not formatted properly." unless BUILD_TIMESTAMP_REGEX.match?(_build_timestamp)
+
+        @build_timestamp = _build_timestamp
+      else
+        raise 'TestTrack failed to load the required build timestamp. ' \
+          'Ensure `testtrack load_build_timestamp` is run and the build timestamp file is present.'
+      end
+    end
+
+    def build_timestamp
+      @build_timestamp.presence || raise('build_timestamp is not defined. Ensure `load_build_timestamp` is run.')
```

Comment:
```suggestion
      @build_timestamp.presence || raise('build_timestamp is not defined because `load_build_timestamp` initializer was not run.')
```